src/Powercord/migrations/20220822-move-themes-plugins.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const { join } = require('path');
+
+module.exports = () => {
+  const oldPlugs = join(__dirname, '..', 'plugins');
+  const newPlugs = join(__dirname, '..', '..', '..', 'plugins');
+  const oldThemes = join(__dirname, '..', 'themes');
+  const newThemes = join(__dirname, '..', '..', '..', 'themes');
```

Comment:
The migration script should check if `oldPlugs` or `newPlugs` actually exist before trying to move them.

If neither exist, do not run the migration.

Reply:
Also don't run if new exists but not old (might be what you meant)

Reply:
`newPlugs` should definitely exist because if you have this script you would have already pulled the changes which add the new plugin folder. I'll add a check for `oldPlugs` though.